requirements/base.txt

```diff
@@ -1,7 +1,7 @@
 # Base dependencies required to run SunPy
 
 numpy>1.7.1
-astropy>=1.3
+astropy>=2.0
 scipy
-pandas>=0.12.0
-matplotlib>=1.1
+pandas>=0.20.0
```

Comment:
Do we actually need to require these? They are supposed to be the minimum possible versions.